### PR TITLE
Added a bunch of alternatives from dryfgy and TomBogle

### PR DESCRIPTION
**Pull request by tombogle**
Added a couple novel questions and improved a few answers

- - -
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/135)

### CODE REVIEW

File: Transcelerator/TxlQuestions.xml [hunk 1]
```diff
@@ -31822,11 +31880,17 @@
...
           <A>He said it to show how he was to die. (33)</A>
+          <Alternative source="TomBogle">Why did Jesus say this?</Alternative>
+          <Alternative source="TomBogle">What was Jesus intending to communicate when he said this?</Alternative>
         </Questions>
         <Questions scrref="JHN 12.34" startref="43012034" endref="43012034">
           <Q>What is the reason that the people were confused?</Q>
           <A>They were confused because the law stated that the Messiah would live forever,</A>
           <A>so how could he say that "the Son of Man must die/be lifted up?" (34)</A>
+          <Alternative source="dryfgy">What is the reason that the people were confused about Jesus' words?</Alternative>
+          <Alternative source="TomBogle">Why the people were confused about Jesus' words?</Alternative>
```
Comment (andrew-polk) [inline Transcelerator/TxlQuestions.xml:31891]: These three alternatives you added are not worded well. I think you intended "Why were..."

**Review by andrew-polk — CHANGES_REQUESTED** [review]

File: Transcelerator/TxlQuestions.xml [hunk 2]
```diff
@@ -62700,6 +62874,16 @@
         <Questions scrref="ROM 10.8-10" startref="45010008" endref="45010010">
           <Q>Connect "being called good" with "getting life" or "being saved".</Q>
           <A>If we believe, and say so in public, immediately God gives us life and calls us good people.</A>
+          <Alternative source="dryfgy">Connect "having faith" and "proclaiming it", with "getting life" or "being saved".</Alternative>
+          <Alternative source="TomBogle">What is the connection between having faith and proclaiming it and being saved?</Alternative>
+          <Alternative source="TomBogle">What is the connection between proclaiming your genuine faith and being saved?</Alternative>
+          <Alternative source="TomBogle">What is the connection between having faith and proclaiming it and "getting life" or "being saved"?</Alternative>
+          <Alternative source="TomBogle">What is the connection between proclaiming your genuine faith and "getting life" or "being saved"?</Alternative>
+          <Alternative source="TomBogle">What does God do when people proclaiming their genuine faith in him?</Alternative>
```
Comment (andrew-polk) [inline Transcelerator/TxlQuestions.xml:62882]: proclaim x2

File: Transcelerator/TxlQuestions.xml [hunk 3]
```diff
@@ -61773,7 +61916,27 @@
         </Questions>
         <Questions scrref="ROM 2.16" startref="45002016" endref="45002016">
           <Q>If all this is so, what need is there to learn all of God's laws?</Q>
-          <A>Unless we accept God's way of saving us through Christ, Christ as judge will condemn us under whatever law we have lived, because none of us in perfect.</A>
+          <A>Unless we accept God's way of saving us through Christ, Christ as judge will condemn us under whatever law we have lived, because none of us is perfect.</A>
+          <Alternative source="dryfgy">So, what does a person really need to be justified and accepted in the sight of God?</Alternative>
+          <Alternative source="TomBogle">On what basis will God judghe people?</Alternative>
```
Comment (andrew-polk) [inline Transcelerator/TxlQuestions.xml:61921]: typo (judghe) x2

File: Transcelerator/TxlQuestions.xml [hunk 4]
```diff
@@ -33804,6 +33932,15 @@
         <Questions scrref="JHN 19.24" startref="43019024" endref="43019024">
           <Q>What is the reason that the soldiers divided Jesus's clothing?</Q>
           <A>They divided Jesus's clothing so that the [Old Testament] scripture was fulfilled. (24)</A>
+          <Alternative source="TomBogle">What is the reason that the soldiers cast lots among themselves for Jesus' clothing?</Alternative>
+          <Alternative source="TomBogle">What is the reason that the soldiers cast lots to divide Jesus' clothing among themselves?</Alternative>
+          <Alternative source="TomBogle">What is the reason that the soldiers cast lots to decide which of them would get to take parts of Jesus' clothing?</Alternative>
+          <Alternative source="TomBogle">What is the reason that the soldiers divided Jesus's clothing among themseleves?</Alternative>
```
Comment (andrew-polk) [inline Transcelerator/TxlQuestions.xml:33938]: typo (themseleves) x2

**Review by andrew-polk — CHANGES_REQUESTED** [review]

File: Transcelerator/TxlQuestions.xml (hunk 1 above)
Comment (tombogle) [reply Transcelerator/TxlQuestions.xml]: Fixed

File: Transcelerator/TxlQuestions.xml (hunk 2 above)
Comment (tombogle) [reply Transcelerator/TxlQuestions.xml]: Fixed

File: Transcelerator/TxlQuestions.xml (hunk 3 above)
Comment (tombogle) [reply Transcelerator/TxlQuestions.xml]: Fixed

File: Transcelerator/TxlQuestions.xml (hunk 4 above)
Comment (tombogle) [reply Transcelerator/TxlQuestions.xml]: Fixed. You inspired me to write a spell-checker. Caught several other errors!

**Review by andrew-polk — COMMENTED** [review]
Looks good assuming you're intentionally switching to American spelling in a few places.

> Reviewed 1 of 2 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/135#-:-O0u_OmSDL9wn6ImdM22:bexpxa5)* status: all files reviewed, 1 unresolved discussion (waiting on @tombogle)

___
*[`spellcheck_xml.py` line 79 at r3](https://reviewable.io/reviews/sillsdev/Transcelerator/135#-O0uWlkw6PaxRLtfUGaV:-O0uWlkw6PaxRLtfUGaW:b9qmokt) ([raw file](https://github.com/sillsdev/Transcelerator/blob/842e9b3024c585741a48312b2ce45652d4353a8b/spellcheck_xml.py#L79)):*
> ```Python
>     args = parser.parse_args()
> 
>     spell_check_xml(args.file_path, args.max_elements, args.output_file)
> ```

I won't pretend I reviewed this.

**Review by andrew-polk — APPROVED** [review]

**Review by andrew-polk — CHANGES_REQUESTED** [review]
> Reviewed 5 of 5 files at r4, 4 of 4 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/135)* status: all files reviewed, 3 unresolved discussions (waiting on @tombogle)

___
*[`Transcelerator/LocalizedPhrases-en-GB.xlf` line 0 at r4](https://reviewable.io/reviews/sillsdev/Transcelerator/135#-O1IRw_e6COSw-UDe2t2:-O1IRw_e6COSw-UDe2t3:bvotgm3) ([raw file](https://github.com/sillsdev/Transcelerator/blob/2df79feb75b431a53799759b01915e726551ed75/Transcelerator/LocalizedPhrases-en-GB.xlf#L0)):*
I didn't review this under the assumption it was all automated and nothing to review (along with the fact that I didn't want to... : ) )
___
*[`Transcelerator/LocalizedPhrases-fr.xlf` line 0 at r5](https://reviewable.io/reviews/sillsdev/Transcelerator/135#-O1N5ejy6LT97JUKfM6B:-O1N5ejy6LT97JUKfM6C:b-i28gf) ([raw file](https://github.com/sillsdev/Transcelerator/blob/7408981556ac3372dc73833a14796bac5badbb08/Transcelerator/LocalizedPhrases-fr.xlf#L0)):*
I can't find any way to review this or the Spanish. Reviewable and github.com won't show it, and my client says it is a binary file and won't show the diff, either.
___
*[`Transcelerator/TxlQuestions.xml` line 0 at r5](https://reviewable.io/reviews/sillsdev/Transcelerator/135#-O1N5puw7CLgRuuiGUvU:-O1N5puw7CLgRuuiGUvV:b-hmiftj) ([raw file](https://github.com/sillsdev/Transcelerator/blob/7408981556ac3372dc73833a14796bac5badbb08/Transcelerator/TxlQuestions.xml#L0)):*
Looks good other than the double p spellings in some new alternates I pointed out on Slack.

**Comment by tombogle** [comment]
*[`Transcelerator/LocalizedPhrases-fr.xlf` line  at r5](https://reviewable.io/reviews/sillsdev/Transcelerator/135#-O1N5ejy6LT97JUKfM6B:-O1Nm3i68kLEaigYZW_F:byef5nk) ([raw file](https://github.com/sillsdev/Transcelerator/blob/7408981556ac3372dc73833a14796bac5badbb08/Transcelerator/LocalizedPhrases-fr.xlf#L)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

I can't find any way to review this or the Spanish. Reviewable and github.com won't show it, and my client says it is a binary file and won't show the diff, either.
</blockquote></details>

You would probably need to pull down both versions and use a diff tool. That's how I attempt to make sure I haven't done something stupid. Ultimately, someone other than you or I should be doing this review, though.

**Review by tombogle — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/135)* status: 4 of 8 files reviewed, 2 unresolved discussions (waiting on @andrew-polk)

___
*[`Transcelerator/TxlQuestions.xml` line  at r5](https://reviewable.io/reviews/sillsdev/Transcelerator/135#-O1N5puw7CLgRuuiGUvU:-O1NmNwCD6cyw-LKDY2F:b-896fix) ([raw file](https://github.com/sillsdev/Transcelerator/blob/7408981556ac3372dc73833a14796bac5badbb08/Transcelerator/TxlQuestions.xml#L)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

Looks good other than the double p spellings in some new alternates I pointed out on Slack.
</blockquote></details>

Done.

**Review by andrew-polk — APPROVED** [review]
> Reviewed 4 of 4 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/135)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @tombogle)

___
*[`Transcelerator/LocalizedPhrases-fr.xlf` line  at r5](https://reviewable.io/reviews/sillsdev/Transcelerator/135#-O1N5ejy6LT97JUKfM6B:-O1NsAIc1FZYL9hoMgHg:b-h59p9n) ([raw file](https://github.com/sillsdev/Transcelerator/blob/7408981556ac3372dc73833a14796bac5badbb08/Transcelerator/LocalizedPhrases-fr.xlf#L)):*
<details><summary><i>Previously, tombogle (Tom Bogle) wrote…</i></summary><blockquote>

You would probably need to pull down both versions and use a diff tool. That's how I attempt to make sure I haven't done something stupid. Ultimately, someone other than you or I should be doing this review, though.
</blockquote></details>

Ok. I'm just going to ignore them unless you want me to do that.

**Pull request merged**